common/views/components/Image/Image.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+
+import {convertImageUri} from '../../../utils/convert-image-uri';
+import {imageSizes} from '../../../utils/image-sizes';
+
+type Props = {|
+  width: string,
+  height?: string,
+  contentUrl: string,
+  clipPathClass?: string,
+  alt?: string,
```

Comment:
Optional?

Reply:
We might want to make it `alt: ?string` so that we purposefully pass `null`, making us feel ashamed (or at least justifying it).

Reply:
Yeah, wasn't sure about this – can't remember why we don't currently have `alt` text on works (but doesn't look like we do, so was mirroring what we currently have).

Will update as per your suggestion.

Reply:
The reason I think is because alt text should not be redundant or provide the same information as text within the context of the image - and the description we have is already on the page